site/docs/v1/tech/apis/_x-fusionauth-tenant-id-required.adoc

```diff
@@ -0,0 +1,11 @@
+==== Request Headers
+
+[.api]
+[field]#X-FusionAuth-TenantId# [type]#[String]# [optional]#Optional#::
+The unique Id of the tenant used to scope this API request.
+The tenant Id is not required on this request when only tenant has been configured.
+If there is more than one tenant, the tenant Id is required.
```

Comment:
Just a small nit, do these each need to be on a s separate line?

Reply:
@robotdan I've started doing that because:

1. if there are no newlines between, Asciidoctor turns them into 1 paragraph.
2. In the future, git blame/seeing what was added when will be a lot easier.

Reply:
I think we should standardize on this, TBH